db/seeds.rb

```diff
@@ -5,3 +5,20 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+10.times do
+  Term.create(name: Faker::Hipster.word)
```

Comment:
Personal opinion: I don't think that Faker::Hipster is funny. But I've been accused of being a hipster myself, soooo  https://media.giphy.com/media/xTiTnsHTXnBheAIM5G/giphy.gif